### PR TITLE
[Cosmos] More docs cleanup

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/cosmos/azure-cosmos/azure/cosmos/database.py [hunk 1]
```diff
@@ -577,16 +577,16 @@ def create_user(self, body, **kwargs):
...
 
         :param body: A dict-like object with an `id` key and value representing the user to be created.
-        The user ID must be unique within the database, and consist of no more than 255 characters.
+         The user ID must be unique within the database, and consist of no more than 255 characters.
         :param request_options: Dictionary of additional properties to be used for the request.
         :param response_hook: a callable invoked with the response metadata
         :returns: A `UserProxy` instance representing the new user.
         :raise `CosmosHttpResponseError`: If the given user couldn't be created.
-        :rtype: ~azure.cosmos.user.UserProxy
+        :rtype: ~azure.cosmos.UserProxy
```
Comment (lmazuel) [inline sdk/cosmos/azure-cosmos/azure/cosmos/database.py]: line 584 should be something like:
```python
:raise ~azure.cosmos.exception.CosmosHttpResponseError: If the given user couldn't be created.

File: sdk/cosmos/azure-cosmos/dev_requirements.txt [hunk 2]
```diff
@@ -1,2 +1 @@
 -e ../../../tools/azure-sdk-tools
--e ../../core/azure-core
```
Comment (lmazuel) [inline sdk/cosmos/azure-cosmos/dev_requirements.txt]: You do need azure-core as dev dep

Comment (annatisch) [reply sdk/cosmos/azure-cosmos/dev_requirements.txt]: Done :)

File: sdk/cosmos/azure-cosmos/azure/cosmos/database.py (hunk 1 above)
Comment (annatisch) [reply sdk/cosmos/azure-cosmos/azure/cosmos/database.py]: Fixed :)

File: sdk/cosmos/azure-cosmos/azure/cosmos/container.py [hunk 3]
```diff
@@ -173,12 +173,12 @@ def read_item(
         :param request_options: Dictionary of additional properties to be used for the request.
         :param response_hook: a callable invoked with the response metadata
         :returns: Dict representing the item to be retrieved.
-        :raise `CosmosHttpResponseError`: If the given item couldn't be retrieved.
+        :raises ~azure.cosmos.errors.CosmosHttpResponseError: If the given item couldn't be retrieved.
         :rtype: dict[str, Any]
 
         .. admonition:: Example:
 
-            .. literalinclude:: ../../samples/examples.py
```
[the remark this answers is not in the document]
Comment (annatisch) [reply sdk/cosmos/azure-cosmos/azure/cosmos/container.py]: Yeah - I believe it needs to be relative to the sdist root. I've built the docs and confirm that the new path is accurate :)

File: sdk/cosmos/azure-cosmos/azure/cosmos/container.py [hunk 4]
```diff
@@ -173,12 +173,12 @@ def read_item(
         :param request_options: Dictionary of additional properties to be used for the request.
         :param response_hook: a callable invoked with the response metadata
         :returns: Dict representing the item to be retrieved.
-        :raise `CosmosHttpResponseError`: If the given item couldn't be retrieved.
+        :raises ~azure.cosmos.errors.CosmosHttpResponseError: If the given item couldn't be retrieved.
```
[the remark this answers is not in the document]
Comment (annatisch) [reply sdk/cosmos/azure-cosmos/azure/cosmos/container.py]: Actually this is a good catch - as I had been trying to consistently remove the "if"s

**Pull request merged**